core/unit_test/CMakeLists.txt

```diff
@@ -414,6 +417,7 @@ IF(Kokkos_ENABLE_CUDA)
       cuda/TestCuda_Crs.cpp
       cuda/TestCuda_WorkGraph.cpp
       cuda/TestCuda_UniqueToken.cpp
+      cuda/TestCuda_ViewResize.cpp
```

Comment:
We should also test this for the hpx backend

Reply:
@Rombur I added the test for the hpx backend. Thanks.